access/api.go

```diff
@@ -67,6 +69,7 @@ func MessageToTransactionResult(message *access.TransactionResultResponse) *Tran
 		StatusCode:   uint(message.StatusCode),
 		ErrorMessage: message.ErrorMessage,
 		Events:       convert.MessagesToEvents(message.Events),
+		BlockID:      flow.HashToID(message.BlockId),
```

Comment:
I guess we return a block ID equal to 0x0..0 when the block is not found, right?

Reply:
yeah all zeroes - that probably needs to be called out in the documentation though - good catch!

Reply:
Yes it would better to mention it. The user may need a way to differentiate a block that is non-found, and a block that is found but its ID turns out to be zeroes (although it's a very rare coincidence).